### DISCUSSION TITLE
CCS from remote cluster?

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#5280] [Unsupported Versions] [answered]
H, is it possible to use CCS on the SO deployment from an already existing elk stack?

**Comment by dougburks** [comment, marked as answer]
You may be able to make this work, but we don't recommend or support it.

**Comment by n0th1nt0lOOz** [comment]
Hi @OzzyKampha 
I have the same problem. Could you share me the way you configure the CCS for this? In my case, SO will perform CCS to my ELK logs system
Thanks

**Reply by OzzyKampha** [reply]
Hi i have not made it work yet… I have not found out how to expose port 9300 in SO.

**Reply by n0th1nt0lOOz** [reply]
you mean you can make port 9300 open for connection from your EKL, right?

**Reply by OzzyKampha** [reply]
Dont know how to open 9300 in SO

**Reply by n0th1nt0lOOz** [reply]
@OzzyKampha  After looking for the so-allow and so-firewall, I think you could add the ELK cluster to searchnode group which allow port 9300
https://docs.securityonion.net/en/2.3/firewall.html
Please update to me. Thanks
